### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: Update README
+
+on:
+  workflow_dispatch:
+  push:
+  schedule:
+    # run at 10am UTC daily
+    - cron: '0 10 * * *'
```
Comment (JSAssassin) [inline .github/workflows/main.yml]: Currently runs daily at 10AM UTC, we might wanna change that. How often do we want the gh workflow to run?

Comment (BigBlueHat) [reply .github/workflows/main.yml]: Yeah, I don't think these specs update that often. Maybe once a week alongside the test result updates (i.e. over the weekend) would be sufficient?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      - name: Setup git config
+        run: |
+          # Setup the username and email. Use 'GitHub Actions Bot' with no email
+          # by default
+          git config user.name "GitHub Actions Bot"
+          git config user.email "<>"
+      - name: Commit
+        run: |
+          # Stage the file, commit and push
+          git add README.md
```
Comment (BigBlueHat) [inline .github/workflows/main.yml:51]: I think this is fine for now, but it sort of tangles the purpose of this repo--it's both the updater...and the updates.

I wonder if we could/should use a `gh-pages` branch and publish both an HTML page (via Jekyll and the README--which should "just work" once you commit there and setup GH Pages on this repo) _and_ publish a JSON file of this data, so we can consume it on the dashboard.

Thoughts?

File: lib/repo-data.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+
+export const repoPairs = [{
```
Comment (BigBlueHat) [inline lib/repo-data.js:5]: If we really wanted to abstract this some more, we could make this a configuration option and pass it into an environment variable via "Settings".

Really just a question of how far we want to take this (at least at this point in time 😄).

File: lib/repo-data.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  spec: 'https://github.com/w3c/vc-di-ecdsa',
+  testSuite: 'https://github.com/w3c/vc-di-ecdsa-test-suite',
```
Comment (BigBlueHat) [inline lib/repo-data.js:7]: I do also wonder about renaming these--or even just making these arrays--so others could (eventually) use this code to compare repos (at least A vs. B) that aren't "spec" nor "testSuite".

File: package.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  "name": "@digitalbazaar/vc-commit-tracker",
+  "version": "0.0.1-0",
+  "description": "Retrieves the most recent commit dates from a list of VC specs and test suites.",
+  "type": "module",
+  "main": "lib/index.js",
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com/"
+  },
```
Comment (BigBlueHat) [inline package.json:11]: @JSAssassin you should add yourself to the `contributors` array: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#people-fields-author-contributors

File: package.json [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/vc-commit-tracker"
+  },
+  "directories": {
+    "lib": "./lib",
+    "templates": "./templates"
+  },
+  "engines": {
+    "node": ">=18"
```
Comment (BigBlueHat) [inline package.json]: The build pipelines use v20, but I guess we know this to work in v18 also?

**Review by BigBlueHat — APPROVED** [review]
Mostly just musings about how far we want to take this and when. 😃 

Once this is in (which can be right away), I'll send in a PR with the open-sourcing bits.

Thanks so much for building this little app!

File: .github/workflows/main.yml (hunk 1 above)
Comment (JSAssassin) [reply .github/workflows/main.yml]: Sounds good. I've updated it to run at 10 AM UTC every Sunday [144b09a](https://github.com/digitalbazaar/vc-commit-tracker/pull/1/commits/144b09a5905155b5daec71c963f3cfae6f2176cd)

File: package.json (hunk 6 above)
Comment (JSAssassin) [reply package.json]: It works in v18 too but for sake of consistency I've now updated `engines.node` to >= 20 [aa588e2](https://github.com/digitalbazaar/vc-commit-tracker/pull/1/commits/aa588e206c8e2ff697b84e12a00e1bc7a5a13645)

File: .github/workflows/main.yml (hunk 2 above)
Comment (JSAssassin) [reply .github/workflows/main.yml:51]: Sounds like a good idea! Thanks for bringing this up. I've opened an issue to track this https://github.com/digitalbazaar/vc-commit-tracker/issues/2, please feel free to update the issue.

File: lib/repo-data.js (hunk 3 above)
Comment (JSAssassin) [reply lib/repo-data.js:5]: Opened issue https://github.com/digitalbazaar/vc-commit-tracker/issues/3

**Pull request merged**